### ISSUE TITLE
Player:NameSpaces Extending 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#231]
Huhhila
Extend player:namespaces elsewhere besides just tp tube channels, starting from digiline channels.

**Comment by S-S-X** [comment]
Difference between teleport tubes and digilines is that there's huge amount of things sending digiline messages and many things have no idea about who is owner, basically a lot of digiline devices are not owned.

This fact makes this very hard and time consuming to implement well and in my opinion results are not really worth the trouble, use cases are extremely narrow and specific to few very special use cases.

Basically current digilines mod API should be rewritten along with all mods that depend on digilines, any idea about overriding existing stuff will eventually either fail or generate significant amounts of extra work to keep things up to date.

Also my personal opinion about feature itself if not thinking about actual implementation problems: bad idea, not needed, not enough use cases.
